alexaapi/request_root.go

```diff
@@ -75,3 +75,10 @@ type Context struct {
 	Viewports   []*Viewport         `json:"Viewports,omitempty"`
 	AudioPlayer *AudioPlayerContext `json:"AudioPlayer,omitempty"`
 }
+
+// Alexa.Presentation.APL
+type APLContext struct {
```

Comment:
This doesn't seem to be used anywhere

Reply:
Not checked in yet, part of the education app